src/pages/Admin/Charity/EditProfile/schema.ts

```diff
@@ -23,7 +23,15 @@ const fileObj = Yup.object().shape<SchemaShape<ImgLink>>({
 //construct strict shape to avoid hardcoding shape keys
 const shape: SchemaShape<FormValues> = {
   categories_sdgs: Yup.array().min(1, "required"),
-  tagline: requiredString,
+  tagline: Yup.string().test(
+    "len",
+    "Maximum length of 140 characters. Required field.",
+    (val) => {
+      return val !== undefined && val.length > 0 && val.length <= 140
```

Comment:
no need to assert `undefined` since form value is always `string` and is set by default in
`EditProfile/index.tsx`

```
tagline: props.tagline,
```

if we are not sure if `tagilne` is `string` - we should update it's definition in `type EndowmentProfile`